src/js/common_shim.js

```diff
@@ -164,6 +164,20 @@ export function shimMaxMessageSize(window) {
       window.RTCPeerConnection.prototype.setRemoteDescription;
   window.RTCPeerConnection.prototype.setRemoteDescription = function() {
     this._sctp = null;
+    // Chrome unilaterally decided to not expose .scpt in plan-b mode.
+    // As usual, adapter.js has to do the 'ugly hack' to cover up the mess.
+    if (browserDetails === 'chrome' && ('sctp' in this)) {
```

Comment:
Should this be negated for sctp?

```suggestion
    if (browserDetails === 'chrome' && !('sctp' in this)) {
```

Reply:
the point is to do this once, even in plan-b mode. But it doesn't work as intended...